provider/image.go

```diff
@@ -629,7 +635,7 @@ func marshalCachedImages(b resource.PropertyValue) ([]string, error) {
 	}
 	c := b.ObjectValue()["cacheFrom"]
 
-	if c.IsNull() {
+	if c.IsNull() || c.IsComputed() {
```

Comment:
Unfortunately, `resource.PropertyValue.IsComputed()` is a footgun.[^1] Basically, `resource.PropertyValue` has two different ways to represent a value as "unknown": `resource.Computed{ ... }` and `resource.Output{ Known: false, ...}`. There can also be secret computed values (which this will not catch).

```suggestion
	if c.IsNull() || c.ContainsUnknowns() {
```

I can't think of a place where it would be correct to use `c.IsComputed()` directly in this provider.

[^1]: https://github.com/pulumi/pulumi/issues/14620